### DISCUSSION TITLE
I need help with unbanning an user who changed their username.

### DISCUSSION THREAD

**Discussion by Keckss** [Epix-Incorporated/Adonis#1556] [Q&A]
Hey! I have a friend who was banned by the Adonis admin system, during his ban he changed his Username (not display name) and now he can not be unbanned. after trying all the unban commands he is still unable to join and we suspect that it was because of the username change. Any way to remove him from the datastore without having to get a new key?

**Comment by hallowxd** [comment]
try unbanning via his userid instead of username

**Comment by AidenJamesYt** [comment]
> Hey! I have a friend who was banned by the Adonis admin system, during his ban he changed his Username (not display name) and now he can not be unbanned. after trying all the unban commands he is still unable to join and we suspect that it was because of the username change. Any way to remove him from the datastore without having to get a new key?

all you gotta do is ;unban userid

**Reply by AidenJamesYt** [reply]
or go into banland (;banland) and edit the ban
